distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc

```diff
@@ -0,0 +1,52 @@
+:title: Validator Plugins on Ingest
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 00
+:summary: Validator plugins run on Ingest
+
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
+depending on whether *Enforce errors* is selected within the admin console. Below is a list
+of the validators run against the data ingested.
+
+[NOTE]
+====
+To navigate to turn on/off *Enforce errors*:
+ . Navigate to the *${admin-console}*.
+. Select the *System* tab.
+. Select the *Configuration* tab.
+. Select *Metacard Validation Marker Plugin*.
+.. If *Enforce errors* is checked, these validators below will be run on ingest.
+.. If *Enforce errors* is not checked, these validators below will *not* be run on ingest.
+====
+
+=== List of Validators run on ingest
+
+* TDF Schema Validation Service
```

Comment:
✏️ We could simplify the syntax of this list by doing 
`TDF Schema Validation Service:: This service validates a TDO against a TDF schema.`

Reply:
fixed